Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -412,7 +391,7 @@ public Task<IUploadProgress> UploadAsync()
 
             try
             {
-                UploadUri = await InitializeUpload(cancellationToken).ConfigureAwait(false);
+                UploadUri = await InitiateSessionAsync(cancellationToken).ConfigureAwait(false);

```

Comment:
Done. However, in doing so, I realized that the existing public naming refers to the session URI as an upload URI: the `IUploadSessionData` interface through which the `ResumeableUpload.UploadSessionData` event reports the URI has an `UploadUri` property. However, the new static creation method I'm adding is named `CreateFromSessionUri` and has a parameter named `sessionUri`. I'd rather use the naming that is present in the docs, but don't want to introduce an inconsistency. What are your thoughts on this?